packages/ses/src/commons.js

```diff
@@ -378,3 +378,36 @@ export const FERAL_STACK_GETTER = feralStackGetter;
  * @type {((newValue: any) => void) | undefined}
  */
 export const FERAL_STACK_SETTER = feralStackSetter;
+
+const getAsyncGeneratorFunctionInstance = () => {
+  // Test for async generator function syntax support.
+  try {
+    // Wrapping one in an new Function lets the `hermesc` binary file
+    // parse the Metro js bundle without SyntaxError, to generate the
+    // optimised Hermes bytecode bundle, when `gradlew` is called to
+    // assemble the release build APK for React Native prod Android apps.
+    // Delaying the error until runtime lets us customise lockdown behaviour.
+    return new FERAL_FUNCTION(
+      'return (async function* AsyncGeneratorFunctionInstance() {})',
+    )();
+  } catch (e) {
```

Comment:
Please spell out `error`, in keeping with the prevailing non-abbreviated aesthetic.
```suggestion
  } catch (error) {
```

Reply:
(note to self to refactor these separately too after)

<img width="426" alt="Screenshot 2024-10-29 at 9 39 10 am" src="https://github.com/user-attachments/assets/04126a5a-02bd-43c4-aca5-226bbef1bf10">